pages/tutor/tutor_course.py

```diff
@@ -0,0 +1,87 @@
+"""Student calendar."""
+
+from pypom import Page, Region
+from selenium.webdriver.common.by import By
+
+
+class TutorCourse(Page):
+    """Tutor course page."""
+
+    _banner_locator = (By.CLASS_NAME, "course-title-banner")
+    _text_book_locator = (By.CLASS_NAME, 'browse-the-book')
+    _past_work_locator = (By.XPATH, "//ul[contains(@class, 'nav')] /li[2]")
+
+    def wait_for_page_to_load(self):
+        """Wait until the page load."""
+        self.wait.until(
+            lambda _: self.find_element(*self._banner_locator).is_displayed())
+
+    @property
+    def weeks(self):
+        """Pass main assignment dashboard."""
+        return self.Weeks(self)
+
+    @property
+    def performance_forcast(self):
+        """Pass performance forcast region."""
+        return self.Performance_forcast(self)
+
+    def past_work(self):
+        """Clicks the past_work tab."""
+        self.find_element(*self._past_work_locator).click()
+        return self
+
+    class Weeks(Region):
+        """The assignemnt dashboard."""
+
+        _this_week_locator = (By.CLASS_NAME, "-this-week")
+        _upcoming_locator = (By.CLASS_NAME, "-upcoming")
+        _this_week_assignments_locator = (By.CSS_SELECTOR, '.-this-week .task')
+        _upcoming_assignments_locator = (By.CSS_SELECTOR, '.-upcoming .task')
+
+        def this_week_assignment(self):
```

Comment:
The function name should be plural because it's returning a list of assignments.